src/adapt/csscasc.js

```diff
@@ -1337,6 +1336,98 @@ adapt.csscasc.CheckConditionAction.prototype.getPriority = function() {
     return 5;
 };
 
+/**
+ * @constructor
+ * @extends {adapt.csscasc.ChainedAction}
+ */
+adapt.csscasc.NegateHelperAction = function() {
+  this.applied = false;
+};
+goog.inherits(adapt.csscasc.NegateHelperAction, adapt.csscasc.ChainedAction);
+
+/**
+ * @override
+ */
+adapt.csscasc.NegateHelperAction.prototype.apply = function(cascadeInstance) {
+  this.applied = true;
+};
+
```

Comment:
Since `NegateHelperAction` has a mutable state, it is better to override `clone` method of `CascadeAction`.